### PR TITLE
led-roulette: Fix some wording and add few tips

### PR DESCRIPTION
**Pull request by Grazfather**
A few tiny grammar mistakes, also added a few tips and comparisons. Feel free to take what you like.

### CODE REVIEW

File: src/05-led-roulette/flash-it.md [hunk 1]
```diff
@@ -127,7 +127,7 @@ Start address 0x8000194, load size 26544
 Transfer rate: 21 KB/sec, 6636 bytes/write.
 ```
 
-And that's it. You'll also new output under the OpenOCD command.
```
Comment (Grazfather) [inline src/05-led-roulette/flash-it.md]: This is one of the important ones to take.

File: src/05-led-roulette/flash-it.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 Flashing is the process of moving our program into the microcontroller's
 (persistent) memory. Once flashed, the microcontroller will executed the flashed
-program everytime is powered on.
+program everytime it is powered on.
```
Comment (Grazfather) [inline src/05-led-roulette/flash-it.md]: Another important one

File: src/05-led-roulette/debug-it.md [hunk 3]
```diff
@@ -118,7 +118,7 @@ disassemble view with the `layout asm` command and advance one instruction at a
 time using `stepi`.
 
 > **NOTE** If you used the `step` command by mistake and GDB got stuck, you can
-> unstuck it by hitting `Ctrl+C`.
+> get unstuck by hitting `Ctrl+C`.
```
Comment (Grazfather) [inline src/05-led-roulette/debug-it.md]: wording.

**Comment by japaric** [comment]
Thanks, @Grazfather. I like all the changes :-).

**Pull request merged**